cmd/bosun/sched/check.go

```diff
@@ -306,6 +310,8 @@ func (s *Schedule) CollectStates() {
 	severityCounts := make(map[string]map[string]int64)
 	abnormalCounts := make(map[string]map[string]int64)
 	ackStatusCounts := make(map[string]map[bool]int64)
+	ackByNotificationCounts := make(map[string]map[bool]int64)
```

Comment:
the map[bool] here is still odd. I think I would rather see ackCounts and unackedCounts maps[string]int